src/app.js

```diff
@@ -33,7 +33,9 @@ app.use(function (err, req, res, _) {
     // set locals, only providing error in development
     res.locals.message = err.message;
     res.locals.error = req.app.get('env') === 'development' ? err : {};
-
+    if (err instanceof ValidationError) {
+        res.status(200).json({ error: String(err) });
```

Comment:
Please add a return keyword here to ensure that only one response is sent in case of ValidationError
```suggestion
        return res.status(200).json({ error: String(err) });
```